lib/v2/xueqiu/user.js

```diff
@@ -44,7 +44,7 @@ module.exports = async (ctx) => {
             const description = item.description + (item.retweeted_status ? `<blockquote>${item.retweeted_status.user.screen_name}:&nbsp;${item.retweeted_status.description}</blockquote>` : '');
             return {
                 title: item.title ? item.title : description.replace(/<[^>]+>/g, ''),
-                description,
+                description: item.text ? item.text : description,
```

Comment:
I would consider this as a regression. Please see the following screenshots for more details.

<details>
<summary>Before this change</summary>

![before](https://user-images.githubusercontent.com/11386903/163856311-5c8de117-2a2f-4b7c-9096-5a1d8b01001a.PNG)
</details>
<details>
<summary>After this change</summary>

![after](https://user-images.githubusercontent.com/11386903/163856413-70712584-75a8-4019-ae06-7c0d5496fefb.PNG)
</details>
<details>
<summary>After this change</summary>

![actual content](https://user-images.githubusercontent.com/11386903/163856493-f8fa041b-1b6c-42e2-8331-c1b70f97428a.PNG)
</details>

Reply:
Sorry, I fix this, because I forgot some item has retweeted.
I also add enough route parameters for testing.
Next time I'd keep in mind that one route fix one PR!